Source/SIMPLib/TestFilters/FilterGroup00.cpp

```diff
@@ -187,3 +189,32 @@ const QString TESTCLASSNAME::getHumanLabel() const
 {
   return "TESTCLASSNAME";
 }
+
+// -----------------------------------------------------------------------------
+FilterGroup00::Pointer FilterGroup00::NullPointer()
+{
+  return Pointer(static_cast<Self*>(nullptr));
+}
+
+// -----------------------------------------------------------------------------
+std::shared_ptr<TESTCLASSNAME> FilterGroup00::New()
+{
+  struct make_shared_enabler : public TESTCLASSNAME
```

Comment:
Use of TESTCLASSNAME instead of FilterGroup00 implies that FilterGroup00 may have been mass replaced by accident.  I strongly suggest looking at the .h / .cpp pair and confirm that everything is as expected.